src/test.cpp

```diff
@@ -35,7 +35,7 @@ void TestSuite::add(Test test) {
     tests[num_tests++] = test;
 }
 
-void TestSuite::run() {
+bool TestSuite::run() {
```

Comment:
Return exit status directly instead of a bool? So we can skipp the wierd if statement... Maybe return the number of failed tests? (Hope we don't go over 255)

Reply:
[There seem to be conventions for exit-codes](http://tldp.org/LDP/abs/html/exitcodes.html) but no rules so that would work, yes. If we have >255 failing tests we have bigger problems than running out of exit codes.